assets/runtime/functions

```diff
@@ -590,6 +590,7 @@ gitlab_configure_oauth_saml_attribute_statements() {
     update_template ${GITLAB_CONFIG} \
       OAUTH_SAML_ATTRIBUTE_STATEMENTS_EMAIL \
       OAUTH_SAML_ATTRIBUTE_STATEMENTS_NAME \
+      OAUTH_SAML_ATTRIBUTE_STATEMENTS_USERNAME \
       OAUTH_SAML_ATTRIBUTE_STATEMENTS_FIRST_NAME \
       OAUTH_SAML_ATTRIBUTE_STATEMENTS_LAST_NAME
     # Remove undefined optional attributes
```

Comment:
cleanup `username:` here, if OAUTH_SAML_ATTRIBUTE_STATEMENTS_USERNAME is not set

Reply:
Yep.

I think this is missing.

```
 exec_as_git sed -i "/username: \\[''\\],/d" ${GITLAB_CONFIG}
```

But also email and name should be removed.

```
 exec_as_git sed -i "/email: \\[''\\],/d" ${GITLAB_CONFIG}
 exec_as_git sed -i "/name: \\[''\\],/d" ${GITLAB_CONFIG}
```

ping @galexrt